### PR TITLE
Fixes minor code style issues

### PR DESCRIPTION
**Pull request by jonkeller**
I copy/pasted some code from these files and @keithwrightbos had some style-related nits, so applying the changes to the original files here also.

### CODE REVIEW

File: 3p/ampcontext.js [hunk 1]
```diff
@@ -100,7 +100,7 @@ export class AbstractAmpContext {
     /** @protected {!IframeMessagingClient} */
     this.client_ = new IframeMessagingClient(win);
     this.client_.setHostWindow(this.getHostWindow_());
-    this.client_.setSentinel(dev().assertString(this.sentinel));
```
Comment (jridgewell) [inline 3p/ampcontext.js]: I would think this is a dev error?

Comment (jonkeller) [inline 3p/ampcontext.js]: I was basing this on feedback from @keithwrightbos at https://github.com/ampproject/amphtml/pull/10596#discussion_r130183444
Keith, do you want to defer to Justin, or...?

Comment (keithwrightbos) [inline 3p/ampcontext.js]: Its an integration question.  Sentinel is expected to be included in the frame.  If its not, it implies the library was used incorrectly (in the wrong location).  If we leave as dev, it gets compiled out and no one is notified.

Comment (lannka) [inline 3p/ampcontext.js]: currently, `user()` does not notify anyone either. i think probably safe to do dev() for now and take actions with real cases.

Comment (jonkeller) [inline 3p/ampcontext.js]: Ok, changed back to dev() here, which now leaves this file as unchanged in this PR. Also made the same change in 3p/iframe-transport-client.js which is new to this PR, but it's the one from #10596 that started the whole conversation.

**Pull request merged**